transformations/disability_transformation/transformation.py

```diff
@@ -0,0 +1,144 @@
+                           "imbeciles":"people with intellectual disabilities", 
+                           "crazy":"person or people with a mental impairment", 
+                           "insane ":"person or people with a mental impairment", 
+                           "wacko":"person with a mental impairment", 
+                           "nuts":"person or people with a mental impairment", 
+                           "retard":"person with an intellectual disability", 
+                           "retards":"people with an intellectual disability", 
+                           "retarded":"person or people with an intellectual disability", 
+                           }
+def postag(text):
+    doc = nlp(text)
+    pos_list = []
+    word_list = []
+    for token in doc:
+        pos_list.append(token.pos_)
+        word_list.append(token.text)
+    print(pos_list, word_list)
+    return word_list, pos_list
+      
+def preserve_capitalization(original, transformed):
+    if original[0].isupper():
+        transformed = transformed.capitalize()
+    else:
+        return original
+    return transformed
+
+def get_index(wl, n):
+    indices = [i for i, x in enumerate(wl) if x == n]
+    return indices
+
+def placement(index_of_dis, wl, pl, input, disability_names, name):
+    text = input.lower()
+    wl,pl = postag(text)
+    index_of_dis = get_index(wl,name)
+    max_len = len(wl)
+
+    for i in index_of_dis: 
+        print("For Occurence", i)
+        print("For Words Less than Maximum Length:")
+        if i < (max_len-1):
+          print("Words in between")
+          if pl[i+1] == 'NOUN': 
+              s = ' '.join(wl)
+              text = s
+          elif pl[i+1]!='NOUN':
+              s = ' '.join(wl) 
+              namew = wl[i]
+              wl[i] = disability_names[namew]
+              text = ' '.join(wl)      
+        elif i >= (max_len-1):
+            print("For Words At Maximum Length:") 
+            namew = wl[i]
+            wl[i] = disability_names[namew]
+            text = ' '.join(wl)
+        else:
+            s = ' '.join(wl)
+            text = s
+        text = preserve_capitalization(input, text)
+    return text   
+
+def replace_punc(text):
+    for i in string.punctuation:
+        text = text.replace(i," "+i)
+    return text  
+
+def restore_punc(text):
+    for i in string.punctuation:
+        text = text.replace(" "+i,i)
+    return text
+
+
+def different_ability(input, disability_names):
+    text = input.lower()
+    text=replace_punc(text)
+    for name in disability_names.keys():
+        if name in text:
+            wl, pl = postag(text)
+            max_len = len(wl)
+            indices = get_index(wl, name)
+            text = placement(indices, wl, pl, input, disability_names, name)         
+        text = preserve_capitalization(input, text)
+        text = restore_punc(text)
+    return text
+
+class DifferentAbilityTransformation(SentenceOperation):
+    tasks = [
```

Comment:
Please add the revelant [keywords](https://github.com/GEM-benchmark/NL-Augmenter/blob/main/docs/keywords.md) for your PR too.